src/commands/profile/create.ts

```diff
@@ -105,16 +98,28 @@ export default class extends Command {
         }
         profileName.trim();
 
-        const blockHttp = new BlockHttp(url);
+        const password = options.password || readlineSync.question('Enter your wallet password: ');
```

Comment:
You could use 

```
OptionsResolver(options,
                'password',
                () => undefined,
                'Enter your wallet password: '),
```
to follow how other parameters are being read (applies to all commands)